now.json

```diff
@@ -65,6 +65,11 @@
     {
       "source": "/page/rozcestnik",
       "destination": "/rozcestnik"
-    }
+    },
+    {
+      "source": "/page/semafor",
+      "destination": "/semafor"
```

Comment:
stranka lokalne s timto nastavenim funguje, ale pokud ji chceme zobrazit v hamburger menu, tak se musi nastavit v admin menu mapotic

Reply:
Jo a navic Zeit chce nejakou extra autorizaci, nejspis kvuli tomuto zmenenemu souboru.

Reply:
Byla tam chyba v JSON formátu. Zeit vyžaduje striktní formát, bez „přebytečných“ čárek na konci seznamu.
Akorát nechápu, proč to Zeit nemůže napsat rovnou do té zprávy k pullrequestu…